src/pkg/zoci/common.go

```diff
@@ -51,6 +46,12 @@ func NewRemote(ctx context.Context, url string, platform ocispec.Platform, mods
 	return &Remote{remote}, nil
 }
 
+// String provides a string representation of the Remote's reference.
+// REVIEW(mkcp): Does this function make sense? I use it for info logging in zoci/copier.go
+func (r *Remote) String() string {
+	return r.Repo().Reference.String()
```

Comment:
I don't feel super strongly, but I'd prefer removing this method and keeping the calls here explicit. `OrasRemote` also contains the manifest and platform so just calling the string method it wouldn't be clear what it will output without looking at the code.

Reply:
Yeah that's fair, this is a bit of a leaky abstraction to try to save a few nested object calls. Deleted in latest.